src/components/UserInfo.vue

```diff
@@ -76,25 +94,55 @@ export default {
   name: "UserInfo",
   data() {
     return {
+      dataReady: false,
       successMessage: "",
+      successState: false,
       errorMessage: "",
-      user: '',
+      errorState: false,
+      user: { 'attributes': {}},
```

Comment:
If the user doesn't have attributes, Keycloak doesn't return any, which results in some `undefined` exceptions. I give it an empty default value to prevent it.